test/test_svi.py

```diff
@@ -1,20 +1,41 @@
 from numpy.testing import assert_allclose
 
-from jax import jit, random
+from jax import jit, random, value_and_grad
 import jax.numpy as np
 from jax.test_util import check_eq
-
+import pytest
 import numpyro
 from numpyro import optim
 import numpyro.distributions as dist
 from numpyro.distributions import constraints
 from numpyro.distributions.transforms import AffineTransform, SigmoidTransform
 from numpyro.handlers import substitute
-from numpyro.infer import ELBO, SVI
+from numpyro.infer import ELBO, SVI, RenyiELBO
 from numpyro.util import fori_loop
 
+@pytest.mark.xfail(run=False)
```

Comment:
I think this is not necessary. Does this test pass in your system: `pytest -vs test/test_svi.py -k renyi`?

Reply:
indeed it does pass! My bad for being too aggressive with the `xfail` !